test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ZookeeperServiceOnK8.java

```diff
@@ -0,0 +1,244 @@
+    }
+
+    private V1beta1ClusterRoleBinding getClusterRoleBinding() {
+        return new V1beta1ClusterRoleBindingBuilder().withKind("ClusterRoleBinding")
+                                                     .withApiVersion("rbac.authorization.k8s.io/v1beta1")
+                                                     .withMetadata(new V1ObjectMetaBuilder()
+                                                                           .withName("default-account-zookeeper-operator")
+                                                                           .build())
+                                                     .withSubjects(new V1beta1SubjectBuilder().withKind("ServiceAccount")
+                                                                                              .withName(NAMESPACE)
+                                                                                              .withNamespace(NAMESPACE)
+                                                                                              .build())
+                                                     .withRoleRef(new V1beta1RoleRefBuilder().withKind("ClusterRole")
+                                                                                             .withName("zookeeper-operator")
+                                                                                             .withApiGroup("rbac.authorization.k8s.io")
+                                                                                             .build())
+                                                     .build();
+    }
+
+    private V1beta1CustomResourceDefinition getZKOperatorCRD() {
+
+        return new V1beta1CustomResourceDefinitionBuilder()
+                .withApiVersion("apiextensions.k8s.io/v1beta1")
+                .withKind("CustomResourceDefinition")
+                .withMetadata(new V1ObjectMetaBuilder().withName("zookeeper-clusters.zookeeper.pravega.io").build())
+                .withSpec(new V1beta1CustomResourceDefinitionSpecBuilder()
+                                  .withGroup(CUSTOM_RESOURCE_GROUP)
+                                  .withNames(new V1beta1CustomResourceDefinitionNamesBuilder()
+                                                     .withKind("ZookeeperCluster")
+                                                     .withListKind("ZookeeperClusterList")
+                                                     .withPlural(CUSTOM_RESOURCE_PLURAL)
+                                                     .withSingular("zookeeper-cluster")
+                                                     .withShortNames("zk")
+                                                     .build())
+                                  .withScope("Namespaced")
+                                  .withVersion(CUSTOM_RESOURCE_VERSION)
+                                  .build())
+                .build();
+
+    }
+
+    private V1beta1ClusterRole getClusterRole() {
```

Comment:
The ZK operator should not require any `ClusterRole`, just a regular `Role`.

Reply:
ok, will update it.
https://github.com/pravega/zookeeper-operator/blob/master/deploy/default_ns/rbac.yaml this has a cluster role for default namespace.